Source/BusinessMomentsAdaptorImage/dataController.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express';
+import rawData from './dataschema';
+
+export let allData = (req: any, res: any) => {
```

Comment:
allData is never reassigned. Use `const` instead.

You can run the following command to get the linting feedback.

```bash
yarn ci
```

Reply:
Fixed